BitwardenShared/Core/Autofill/Services/AutofillCredentialServiceTests.swift

```diff
@@ -764,17 +764,4 @@ class AutofillCredentialServiceTests: BitwardenTestCase { // swiftlint:disable:t
 
         XCTAssertEqual(errorReporter.errors as? [BitwardenTestError], [.example])
     }
-
-    // MARK: Private
-
-    /// Waits until `identityStore.removeAllCredentialIdentitiesCalled` is `true` and then resets it
-    /// to `false`. This happens because of the first value of `vaultTimeoutService.vaultLockStatusSubject`
-    /// which is `nil` and removes all credentials when the test is setup.
-    private func waitAndResetRemoveAllCredentialIdentitiesCalled() async throws {
-        try await waitForAsync {
-            self.identityStore.removeAllCredentialIdentitiesCalled
-        }
-
-        identityStore.removeAllCredentialIdentitiesCalled = false
-    }
```

Comment:
🤔 Did the SUT code change? Because I remember having to add this in order for the tests not to fail occasionally on CI.

Reply:
I moved this into `setUp()` so it would apply to all tests in the file.

Reply:
Ohh missed that was in the setup, niceeee love it! Thanks!